src/spikeinterface/preprocessing/filter.py

```diff
@@ -56,6 +57,10 @@ class FilterRecording(BasePreprocessor):
         - numerator/denominator : ("ba")
     ftype : str, default: "butter"
         Filter type for `scipy.signal.iirfilter` e.g. "butter", "cheby1".
+    causal_mode : bool, default: False
+        If true, filtering is applied in just one direction.
+    direction : "forward" | "backward", default: "forward"
+        when causal_mode = True, defines the direction of the filtering
```

Comment:
Why not having direction : "forward" | "backward" | "forward-backward" ?
That would avoid causal_mode no ?

Reply:
That is actually true, makes sense to me.